libraries/AP_DroneCAN/AP_DroneCAN.cpp

```diff
@@ -153,6 +157,16 @@ const AP_Param::GroupInfo AP_DroneCAN::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("ESC_RV", 9, AP_DroneCAN, _esc_rv, 0),
 
+#if AP_RELAY_DRONECAN_ENABLED
+    // @Param: RLY_RT
+    // @DisplayName: Relay output rate
```

Comment:
If this is a dronecan specific parameter maybe it should have DroneCAN at the beginning of the display name.

Reply:
done, "DroneCAN relay output rate"